### ISSUE TITLE
Clip PCA lines so they don't overwrite scroll bars

### ISSUE THREAD

**Issue by pont-us** [pont-us/PuffinPlot#167]
If the Zplot is at the bottom or right edge of the main window, and  the scroll bar is shown, the PCA lines (if shown) overwrite the scroll bar. At present, the PCA lines are explicitly clipped to the ZPlot's area. They also need to be clipped to the window's drawing area (excluding the scroll bars) in order to avoid messing up the window furniture.

**Comment by pont-us** [comment]
```
Bugs Everywhere data:
Created at: 2011-11-09 03:26:57
Status: fixed
Severity: minor
UUID: 6933830e-c515-4968-ba65-6e80cbc95389
Short name: a88/6933
Ditz bug index: 165
```

**Comment by pont-us** [comment]
2012-05-17 11:12:44

This got fixed as a side effect of closing #221: Configurable PCA fit line

UUID: 5c8839f4-36fb-44be-a085-b2b7c0319cfa
